zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -553,8 +559,8 @@ mod tests {
             SELECT transactions.txid                   AS txid,
                    2                                   AS output_pool,
                    sapling_received_notes.output_index AS output_index,
-                   sent_notes.from_account             AS from_account,
-                   sapling_received_notes.account      AS to_account,
+                   sent_notes.from_account_id          AS from_account_id,
+                   sapling_received_notes.account_id   AS to_account_id,
```

Comment:
These changes need to be noted in the CHANGELOG, as this view is part of the public API.